src/components/jobs/JobItem.jsx

```diff
@@ -23,15 +24,12 @@ const Job = props => (
 
 const JobWrapper = styled.div`
   &&& .card {
-    display: grid;
     height: 100%;
+    padding: 10px;
   }
   .content-container {
     display: grid;
-    grid-template-rows: 150px 5% 50%;
-    grid-template-columns: 100%;
-    row-gap: 5px;
-    grid-auto-flow: row;
+    grid-template-rows: 150px auto auto;
```

Comment:
When you move the image out of the <Card.Content> this grid shouldn't be necessary.

Reply:
This most definitely does not work :)

Reply:
And why is that?

Reply:
It breaks the whole inside card layout for some reason. For now, I've decided with @emielvanrijn that we leave the css of the insides of the cards as it is now. The joblist is now a semantic grid, other homogeneous grids are in the progress of being converted as well.